app/uk/gov/hmrc/traderservices/views/templates/GovukLayoutWrapper.scala.html

```diff
@@ -51,6 +51,15 @@
  <meta http-equiv="refresh" content="@refresh"/>
 </noscript>
 }
+
+  <!-- TODO this <meta> should only be included when the user is logged in -->
+  <!-- TODO set correct timeout (900), countdown (780) and keep-alive-url -->
+  <meta name="hmrc-timeout-dialog" content="hmrc-timeout-dialog"
+  data-language="en"
+  data-timeout="@appConfig.timeout"
+  data-countdown="@appConfig.countdown"
+  data-keep-alive-url="/trader-services/keep-alive"
```

Comment:
these URLs shall not be hardcoded, pls inject Routes and reference the routes

Reply:
Sure thing, it's a placeholder URL for now, as it's still WIP